### PR TITLE
Fix inheritance code generation

### PR DESCRIPTION
**Pull request by soutaro**
This PR is to let interface classes with inheritance handle parameters correctly.

`VersionedTextDocumentIdentifier` would explain the problem:

```rb
      class VersionedTextDocumentIdentifier < TextDocumentIdentifier
        def initialize(uri:, version:)
          @attributes = {}

          @attributes[:version] = version       # uri is ignored! 🗑️ 

          @attributes.freeze
        end
    ...
```

The new generate code set up the parents' attributes: call `super`, `dup` the `@attributes`, and finally set the attributes of the children.

### CODE REVIEW

File: scripts/generateFiles.ts [hunk 1]
```diff
@@ -195,7 +195,16 @@ module LanguageServer
...
+            {{#each definition.parent.members}}
+            {{name}}: {{local_var name}},
+            {{/each}}
+          )
+          @attributes = @attributes.dup
+          {{else}}
           @attributes = {}
+          {{/if}}
 
           {{#each definition.members}}
```
Comment (soutaro) [inline scripts/generateFiles.ts:209]: Rewriting this `definition.members` to `definition.allMembers` would be a simpler solution for this...

Comment (mtsmfm) [reply scripts/generateFiles.ts:209]: I think it should be `definition.allMembers` :bow:

**Comment by soutaro** [comment]
@mtsmfm It now uses `allMembers`!

**Comment by mtsmfm** [comment]
Can you add a test to make sure it works?

Like the following:

```diff
+  def test_inheritance
+    identifier = LSP::Interface::VersionedTextDocumentIdentifier.new(
+      uri: "http://example.com",
+      version: 1
+    )
+
+    assert { identifier.attributes == {uri: "http://example.com", version: 1} }
+  end
+
```

File: test/language_server/protocol_test.rb [hunk 2]
```diff
@@ -73,4 +73,13 @@ def to_jsonrpc(hash)
 
     "Content-Length: #{hash_str.bytesize}\r\n" + "\r\n" + hash_str
   end
+
+  def test_sub_interface_set_super_attributes
```
Comment (soutaro) [inline test/language_server/protocol_test.rb:77]: @mtsmfm I'm not very sure if the test makes sense...

**Comment by mtsmfm** [comment]
@soutaro I've just released 3.14.0.2!

**Pull request merged**